### DISCUSSION TITLE
Praise

### DISCUSSION THREAD

**Discussion by anderer** [automadcms/discussions#25] [Show and tell]
i am a total laymen - but i love to tinker with websites. so what i do is try'n'error until a website looks like i want it to look which typically is far off from the regular template.

In 2009 i started with a wordpress website for a project from work, it ran until 2013 when I replaced it with mediawiki, which is still running out in the wild but rather outdated/uncared for since 2017. parallel i started a site for a different project in 2019 based in GRAV and since that time i am running also three discourse platforms for three different projects. and in addition, i have to work with typo3 for the regular website of my work - to put up news etc.

today, i went forward exploring automad V2. this exploration started yesterday - and to see what it can do, i just created the same layout of one of my existing projects - and now after four hours I am mostly done - i just need to fill up the text and then i have an automad copy of it - to be honest, i don't want to move back to the old type. i need to discuss it with a colleague of mine, but i guess we will switch the technology to automad some time soon. be it in alpha/beta/stable doesn't matter.

so before this background i have to say that the automad V2 is just a lot of fun to work with. thanks!

**Comment by marcantondahmen** [comment]
Thank you!
